emsdk.py

```diff
@@ -2587,6 +2592,15 @@ def expand_sdk_name(name):
   elif name in ('latest-upstream', 'latest-clang-upstream', 'latest-releases-upstream'):
     return str(find_latest_releases_sdk('upstream'))
   elif name in ('tot', 'sdk-tot', 'tot-upstream'):
+    if activating:
+      # When we are activating a tot release, assume that the currently
+      # installed SDK, if any, is the tot release we want to activate.
+      # Without this `install tot && activate tot` will race with the builders
+      # that are producing new builds.
```

Comment:
What if the user does `emsdk install tot ; emsdk install 2.0 ; emsdk activate tot` - that would activate 2.0? So `activate tot` is a little misleading.

If we want to change it this way, perhaps it should be just `activate` without specifying `tot` (not immediately, but deprecate the `tot`)?

Or, we could save the last installed `tot` version in a file, and activate that. That seems safer to me.

Reply:
Sure.. I did think of doing that but decided it probably wan't worth it.

Since we can only have one sdk installed a time the whole idea of activation make little sense with today's "only one SDK at a time" approach.  

Do you think we need to worry about folks who run `emsdk activate tot` without running `emsdk install tot` first?

Reply:
(I actually miss the ability to install many SDKs and activate just one of them).

Reply:
I forgot we only allow one at a time... we did at least fix things to avoid repeated downloads, but not installs, then?

If we only allow one then `emsdk activate` seems unnecessary, and maybe we can remove it, unless we do want to support multiple installs again.